api/users.rb

```diff
@@ -11,11 +11,54 @@
   end
 end
 
+get "#{APIPREFIX}/users/:course_id/stats2" do |course_id|
```

Comment:
This is a temporary name to allow testing both endpoints. This will replace stats.